internal/proxy/meta_cache.go

```diff
@@ -1108,6 +1108,13 @@ func (m *MetaCache) RefreshPolicyInfo(op typeutil.CacheOp) (err error) {
 			return err
 		}
 
+		if resp.GetStatus().GetErrorCode() != commonpb.ErrorCode_Success {
```

Comment:
```suggestion
		if !merr.Ok(resp.GetStatus()) {
```

Reply:
or you could just use 
```suggestion
		if err := merr.CheckRPCCall(resp, err); err != nil {
```
mergin previous error check block

Reply:
Thanks for comments. Changed the code to the first suggestion since the second one may lose some invalid response err msg.